submarine-server/server-submitter/submitter-k8s/src/main/java/org/apache/submarine/server/submitter/k8s/K8sJobSubmitter.java

```diff
@@ -186,6 +197,80 @@ private Job parseResponseObject(Object object, ParseOp op) throws SubmarineRunti
     throw new SubmarineRuntimeException(500, "K8s Submitter parse upstream response failed.");
   }
 
+  public String getLog(final Job job) {
+    if (job == null) {
+      return null;
+    }
+    final CoreV1Api COREV1_API = new CoreV1Api(client);
+
+    try {
+      final V1PodList podList = COREV1_API.listNamespacedPod(
+          job.getSpec().getSubmitterSpec().getNamespace(),
+          "false", null, null,
+          getJobLabelSelector(job), null, null,
+          null, null);
+
+      final V1Pod pod = podList.getItems().get(0);
```

Comment:
It would be several containers in the "podList", here it only gets the first one. Is it better that we merge all container's pod with a format? Like this:
{
jobId: job_123,
jobType: tensorflow,
LogContent: [
{podName: tf-ps-0,
podLog: 123123123},
{podName: tf-worker-0,
podLog: 456456456},
...
]
}

Reply:
Ok, I will modify it